powerdns_exporter.go

```diff
@@ -55,13 +56,55 @@ type ServerInfo struct {
 	ZonesUrl   string `json:"zones_url"`
 }
 
-// StatsEntry is used to parse JSON data from 'server/localhost/statistics' endpoint
+// StatsEntry contains the Item property which contains the actual item
 type StatsEntry struct {
+	Item interface{}
+}
+
+// StatisticItemEntry holds values of ring and map statistic items
+type StatisticItemEntry struct {
```

Comment:
sug: rename to `StatisticCollectionItemEntry` to make the connection to `StatisticCollectionItem` more explicit